doc/man/man3/crypto_kex_x_init_client.3monocypher

```diff
@@ -176,17 +192,18 @@ connection with each other.
 .Pp
 Client:
 .Bd -literal -offset indent
-crypto_kex_ctx client_ctx; /* key exchange context */
-uint8_t sk[32];            /* 32 random bytes long-term secret key */
-uint8_t their_pk[32];      /* the server's public key
+crypto_kex_client_ctx client_ctx; /* key exchange context */
+uint8_t client_sk[32];     /* 32 random bytes long-term secret key */
+uint8_t server_pk[32];     /* the server's public key
                             * (known ahead of time) */
 uint8_t seed[32];          /* 32 random bytes */
 uint8_t buf[80];           /* buffer for the key exchange messages */
 uint8_t session_key[32];   /* the resulting session key */
 int fd;                    /* the socket of the connection */
 
 /* (generate random bytes in seed) */
-crypto_kex_x_init_client(&client_ctx, seed, sk, NULL, their_pk);
+crypto_kex_x_init_client(&client_ctx, seed, client_sk, NULL,
+        server_pk);
```

Comment:
alignment doesn't look perfect, did you mean the following?

    crypto_kex_x_init_client(&client_ctx, seed, client_sk, NULL,
                             server_pk);